packages/liveblocks-react/src/factory.tsx

```diff
@@ -109,14 +136,93 @@ function makeMutationContext<
   };
 }
 
+type Options<TUserMeta extends BaseUserMeta> = {
+  /**
+   * @beta
+   *
+   * An asynchronous function that returns user info from a user ID.
+   */
+  resolveUser?: (
+    options: ResolveUserOptions
+  ) => Promise<TUserMeta["info"] | undefined>;
+
+  /**
+   * @beta
+   *
+   * An asynchronous function that returns a list of user IDs matching a string.
+   */
+  resolveMentionSuggestions?: (
+    options: ResolveMentionSuggestionsOptions
+  ) => Promise<string[]>;
+
+  /**
+   * @internal Internal endpoint
+   */
+  serverEndpoint?: string;
+};
+
+let hasWarnedIfNoResolveUser = false;
+
+function warnIfNoResolveUser(usersCache?: AsyncCache<unknown, unknown>) {
+  if (
+    !hasWarnedIfNoResolveUser &&
+    !usersCache &&
+    process.env.NODE_ENV !== "production"
+  ) {
+    console.warn(
+      "Set the resolveUser option in createRoomContext to specify user info."
+    );
+    hasWarnedIfNoResolveUser = true;
+  }
+}
+
+let hasWarnedAboutCommentsBeta = false;
+function warnIfBetaCommentsHook() {
+  if (!hasWarnedAboutCommentsBeta && process.env.NODE_ENV !== "production") {
+    console.warn(
+      "The comments API is still in beta and may change in the future. Please report any issues you encounter at https://github.com/liveblocks/liveblocks/issues/new/choose"
+    );
```

Comment:
The following warning is displayed (once) when any of the following hooks are used:

- `useThreads`
- `useThreadsSuspense`
- `useCreateThread`
- `useEditThreadMetadata`
- `useCreateComment`
- `useEditComment`
- `useDeleteComment`

I am using `console.warn` but it seems the behavior of console.warn()—including whether it shows a stack trace or not—can vary between different environments (e.g., browsers, Node.js, etc.) and is not directly configurable using standard JavaScript or APIs. Any thoughts on this @nvie @marcbouchenoire ?

```
Warning: The Comments API is still in beta and may change in the future. Please report any issues you encounter at https://github.com/liveblocks/liveblocks/issues/new/choose 
```

Reply:
I am open to replacing the console message to say something else (Add link to docs, etc). Open to feedback @marcbouchenoire @nvie @FlowFlorent

Reply:
I tweaked the message to give context to non-beta users who would try to use Comments instead.